community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/RelationshipGroupRecord.java

```diff
@@ -85,8 +86,9 @@ public RelationshipGroupRecord initialize( boolean inUse, int type,
     @Override
     public void clear()
     {
-        initialize( false, -1, NO_NEXT_RELATIONSHIP.intValue(), NO_NEXT_RELATIONSHIP.intValue(),
-                NO_NEXT_RELATIONSHIP.intValue(), NO_NEXT_RELATIONSHIP.intValue(), NO_NEXT_RELATIONSHIP.intValue() );
+        initialize( false, NULL_REFERENCE.intValue(), NO_NEXT_RELATIONSHIP.intValue(),
```

Comment:
hm should we use either `NULL_REFERENCE` or `NO_NEXT_RELATIONSHIP` everywhere then?